lib/ssh_lib.py

```diff
@@ -66,3 +67,35 @@ def copy_file_to_host(host, local_file_path, destination_path):
 
     sftp.put(local_file_path, destination_path)
     sftp.close()
+
+
+def add_ssh_keys_to_instances(instances):
+    """
```

Comment:
For docstrings, let's stick to the same convention as we have in other parts of the code.
Example here:
https://github.com/osbuild/cloud-image-val/blob/3bd776f5afe5b6bda0a79be12cdd1535994684b4/lib/test_lib.py#L27

Reply:
Ack